internal/rbd/group/group_snapshot.go

```diff
@@ -69,6 +71,12 @@ func GetVolumeGroupSnapshot(
 
 	attrs, err := vgs.getVolumeGroupAttributes(ctx)
 	if err != nil {
+		if errors.Is(err, librbd.ErrNotFound) {
```

Comment:
why we can get this error condition met? i don't think `getVolumeGroupAttributes` directly return librbd error can you please check on this one again?

Reply:
We are adding the `librbd.ErrNotFound` error to `ErrRBDGroupNotFound` error here: https://github.com/ceph/ceph-csi/blob/c7d54ab7767e04d597e891d6307249dd5a8cec79/internal/rbd/group/volume_group.go#L37, therefore, this check would work (I have tested as well)